### ISSUE TITLE
Create special.external test jobs

### ISSUE THREAD

**Issue by smlambert** [AdoptOpenJDK/openjdk-tests#1700]
We have tagged external/functional-test and external/system-test targets as special, and will use those tests which run the functional tests and system tests in Docker containers as the way to verify whether unofficial Docker images are 'good' to move into the official Docker hub repo.

We will need special.external test jobs for all active jdk versions (8,11,14,15), both impls we build (hs, j9), most architectures we produce images for (start with x86,s390x,ppcle64).

These jobs (like Grinders and all test pipelines) can take parameters DOCKER_REQUIRED=true and DOCKERIMAGE_TAG (which will default to ubuntu, but can also take ubi, ubi-minimal, alpine, clefos, centos, debian, debianslim).  Ideally, we cycle through each of those OSes over the course of regular testing.  Will first enable with default ubuntu.

**Comment by smlambert** [comment]
| jdk version | impl | DOCKERIMAGE_TAG sample  | Job link per arch |
| ------------- | ------------- | ------------- | ------------- | 
| 8 | hs  | ubuntu | xlinux, plinux, zlinux | 
| 8 | j9 | ubi |  | 
| 11 | hs |  alpine |  | 
| 11 | j9 |  debianslim |  | 
| 14 | hs | ubi-minimal |  | 
| 14 | j9 | debian |  |

**Comment by james-crowley** [comment]
@smlambert Logs for `cmdLineTester_SCCommandLineOptionTests_1`. Seems like two test failed. 

```
---TEST RESULTS---
11:14:43  Number of PASSED tests: 25 out of 27
11:14:43  Number of FAILED tests: 2 out of 27
11:14:43  
11:14:43  ---SUMMARY OF FAILED TESTS---
11:14:43  nameOption2
11:14:43  nameOption4
```

**Respective Tests**
nameOption2:
```
11:14:35  Testing: nameOption2
11:14:35  Test start time: 2020/04/01 15:14:34 Greenwich Mean Time
11:14:35  Running command: sh nameOption2.sh /opt/java/openjdk/bin
11:14:35  Time spent starting: 2 milliseconds
11:14:35  Time spent executing: 647 milliseconds
11:14:35  Test result: FAILED
11:14:35   [OUT] Hello, world!
11:14:35   [OUT] TEST PASSED
11:14:35   [OUT] nameOption2: TEST FAILED
11:14:35   [OUT] nameOption2: Expected to find the cache name
11:14:35   [OUT] TEST PASSED
11:14:35   [ERR] JVMSHRC806I Compressed references persistent shared cache "root" has been destroyed. Use option -Xnocompressedrefs if you want to destroy a non-compressed references cache.
11:14:35  >> Success condition was found: [Output match: TEST PASSED]
11:14:35  >> Failure condition was found: [Output match: TEST FAILED]
11:14:35  >> Failure condition was not found: [Output match: Error:]
11:14:35  >> Failure condition was not found: [Output match: Unhandled Exception]
11:14:35  >> Failure condition was not found: [Output match: Exception:]
```

nameOption4:
```
11:14:36  Testing: nameOption4
11:14:36  Test start time: 2020/04/01 15:14:36 Greenwich Mean Time
11:14:36  Running command: sh nameOption4.sh /opt/java/openjdk/bin
11:14:36  Time spent starting: 1 milliseconds
11:14:36  Time spent executing: 596 milliseconds
11:14:36  Test result: FAILED
11:14:36   [OUT] Hello, world!
11:14:36   [OUT] TEST PASSED
11:14:36   [OUT] nameOption4: TEST FAILED
11:14:36   [OUT] nameOption4: Expected to find the cache name
11:14:36   [OUT] TEST PASSED
11:14:36   [ERR] JVMSHRC806I Compressed references persistent shared cache "_root_root" has been destroyed. Use option -Xnocompressedrefs if you want to destroy a non-compressed references cache.
11:14:36  >> Success condition was found: [Output match: TEST PASSED]
11:14:36  >> Failure condition was found: [Output match: TEST FAILED]
11:14:36  >> Failure condition was not found: [Output match: Error:]
11:14:36  >> Failure condition was not found: [Output match: Unhandled Exception]
11:14:36  >> Failure condition was not found: [Output match: Exception:]
```

**Comment by smlambert** [comment]
Related to this: https://github.com/AdoptOpenJDK/openjdk-docker/issues/236

**Comment by smlambert** [comment]
Closed via #1795
